.github/CODEOWNERS

```diff
@@ -504,6 +504,11 @@ x-pack/examples/files_example @elastic/kibana-app-services
 
 /x-pack/plugins/security_solution/server/lib/detection_engine/routes/fleet @elastic/security-detections-response-rules
 /x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/create_rule_exceptions_route* @elastic/security-solution-platform
+/x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/find_rule_exceptions_route* @elastic/security-solution-platform
+/x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/import_rules_route* @elastic/security-solution-platform
+/x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/preview_rules_route* @elastic/security-detections-response-alerts
+/x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils @elastic/security-solution-platform
```

Comment:
I'm fine with this workaround if it saves us some time short-term (like, this week), but long-term this approach is not really scalable and hardly maintainable. We won't be able to keep up with every file being added and reflect this in the CODEOWNERS file.

I think we will be able to move stuff from the `/x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules` folder to subdomains very soon (On Week!) as part of our subdomains efforts (https://github.com/elastic/kibana/issues/138600).